assets/js/search.js

```diff
@@ -0,0 +1,126 @@
+;(function () {
+    'use strict'
```

Comment:
Do we have a linter which would yell at me? Otherwise I'd suggest to introduce one (ESLint). Shall I open a ticket for that?